crates/orderbook/src/database/total_surplus.rs

```diff
@@ -15,6 +15,21 @@ WITH regular_orders AS (
 onchain_orders AS (
     SELECT ARRAY_AGG(uid) AS ids FROM onchain_placed_orders WHERE sender = $1
 ),
+all_orders AS (
+    -- Get orders from the orders table that match the owner criteria
+    SELECT uid, kind, buy_amount, sell_amount, fee_amount, buy_token, sell_token
+    FROM orders
+    -- use this weird construction instead of `where owner=address or sender=address` to help postgres make efficient use of indices
+    WHERE uid = ANY(ARRAY_CAT((SELECT ids FROM regular_orders), (SELECT ids FROM onchain_orders)))
+    
+    UNION ALL
+    
+    -- Get orders from the jit_orders table that match the owner criteria but are not in orders
+    SELECT j.uid, j.kind, j.buy_amount, j.sell_amount, j.fee_amount, j.buy_token, j.sell_token
+    FROM jit_orders j
+    LEFT JOIN orders o ON j.uid = o.uid
+    WHERE j.owner = $1 AND o.uid IS NULL
```

Comment:
Joining the tables seems to be redundant here.
```suggestion
    WHERE j.owner = $1 
      AND NOT EXISTS (
            SELECT 1 
            FROM orders o 
            WHERE o.uid = j.uid
        )
```

Reply:
Took `AND NOT EXISTS` since the `NOT IN` is slow